src/GraphQL/Execution/ExecutionOptions.cs

```diff
@@ -73,5 +73,7 @@ public class ExecutionOptions : IProvideUserContext
         /// from your dependency injection framework.
         /// </summary>
         public IServiceProvider? RequestServices { get; set; }
+
+        public Inputs? InputExtensions { get; set; }
```

Comment:
See  https://github.com/graphql/graphql-over-http/blob/main/spec/GraphQLOverHTTP.md#request-parameters:

query -> Query [OK]
operationName -> OperationName [OK]
variables -> Inputs [rename to Variables]
extensions -> InputExtensions [rename to Extensions]

?

Reply:
`Inputs` class should not be renamed, only properties

Reply:
i agree with those suggestions

Reply:
just for clairifcation, `ExecutionOptions` will have just `Extensions`, but `IResolveFieldContext` will have `InputExtensions` and `OutputExtensions`?  Seems maybe we should keep `InputExtensions` named the same throughout.

Reply:
Yes, `IResolveFieldContext` will have `InputExtensions` and `OutputExtensions`. `ExecutionOptions` essentially is an "input configuration", only input.  I would keep naming of properties closed to spec - `Extensions` for `ExecutionOptions`. Context already implies "input" interpretation.